docs(GcpRedisInstance,AwsRedisInstance): update docs (#734)
kyma-project/cloud-manager: docs/user/resources/04-70-aws-redis-instance.md

When creating AwsRedisInstance, there is only one mandatory field: `cacheNodeType`.
It specifies the underlying machine that will be used for the cache.

> As in-transit encryption is always enabled, communication with Redis instance requires a trusted CA (Certificate Authority). It should be installed on the container (e.g. via `apt-get install -y ca-certificates && update-ca-certificate`).

Reviewer: ```suggestion
As in-transit encryption is always enabled, communication with the Redis instance requires a trusted Certificate Authority (CA). You must install it on the container (e.g., using `apt-get install -y ca-certificates && update-ca-certificate`).
```
